docs/sources/installation/helm/install-scalable/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Install the Helm Chart
+menuTitle: Install Loki
+description: Install Loki in scalable mode.
+aliases:
+  - /docs/writers-toolkit/latest/templates/task-template
+weight: 100
+keywords:
+  - keyword
+  - key
+  - word
+---
+
+# Install the Helm Chart
+<!-- vale Grafana.Quotes = NO -->
+<!-- vale Grafana.Quotes = YES -->
+
+This Helm Chart installation runs the Grafana Loki cluster within a Kubernetes cluster.
+
+By default, this chart configures Loki to run `read` and `write` targets in a [scalable](../../fundamentals/architecture/deployment-modes/#simple-scalable-deployment-mode), highly available architecture (3 replicas of each) designed to work with AWS S3 object storage. It will also configure meta-monitoring of metrics and logs.
```

Comment:
again, maybe want to be clear that "default" here mean "default when using object storage" or something similar.